virttest/utils_misc.py

```
@@ -4698,3 +4695,79 @@ def _md5(fd):
+    """
+    Verify if specific sev feature is enabled inside the VM
+
+    :param session: session object to vm
+    :param params: Dictionary with the test parameters
+    :param vm: virtual machine.
+    """
+    cvm_type = params["vm_secure_guest_type"]
+    try:
+        cvm_guest_info = vm.monitor.query_sev()
+        if not cvm_guest_info:
+            raise exceptions.TestFail("QMP query-sev returned empty response.")
```

Ruff (0.14.8)

Abstract `raise` to an inner function

(TRY301)